### ISSUE TITLE
missing notes in export after time change

### ISSUE THREAD

**Issue by Bacchushlg** [Audiveris/audiveris#265]
[Manha de Carneval.zip](https://github.com/Audiveris/audiveris/files/3008450/Manha.de.Carneval.zip)

Exporting this score leads to a mysterious phenomenon on the 4th page: after the time change (3/8) every 2nd measure has lost it's first note of the top voice. Inside Audiveris everything looks fine.

**Comment by hbitteur** [comment]
I could not reproduce the problem exactly.
If I ask the page for rhythm reprocessing (using right-click popup menu, Page #1, reprocess page rhythm), the 3/8 value is correctly taken into account. And the MusicXML export is OK.

However, I noticed undue messages on measures 14, 16 and 18, though the resulting rhythm was correct on these measures. I fixed this in commit 2f09f4f76cfd9b114808b444b82ba3afbb275a20 "Improvements in the building of time slots" in development branch.
